addon/adapters/github.js

```diff
@@ -19,7 +19,8 @@ export default RESTAdapter.extend({
   }),
 
   pathForType(type) {
-    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')));
+    return Ember.String.camelize(Ember.String.pluralize(type.replace('github', '')))
+      .replace('organizations', 'orgs');
```

Comment:
This is a bit of a work in progress. We can back it out if you like.

I was trying to consolidate adapters as much as possible and this was the only transformation in the `organizations` adapter. It's really about the path for the type. I considered conditionalizing it and decided it would be pretty much the same either way.